src/js/background.js

```diff
@@ -843,6 +865,14 @@ Badger.prototype = {
     return this.getSettings().getItem("checkForDNTPolicy");
   },
 
+  isAlternateErrorPagesEnabled: function() {
```

Comment:
These are unused. Also unnecessary as a rule: look up from `getSettings()` directly where you need it unless we do the lookup enough to justify its own function.